ivy_tests/test_ivy/test_functional/test_core/test_statistical.py

```diff
@@ -82,7 +82,20 @@ def _statistical_dtype_values(draw, *, function, min_value=None, max_value=None)
                 | helpers.floats(min_value=0, max_value=max_correction - 1)
             )
         return dtype, values, axis, correction
-    return dtype, values, axis
+
+    if isinstance(axis, tuple):
+        axis = axis[0]
+
+    where_shape = draw(
+        helpers.mutually_broadcastable_shapes(
+            num_shapes=1, base_shape=shape, min_dims=0, max_dims=axis
```

Comment:
I have used `min_dims=0` as the default value defined is 1 which was causing errors as the axis would take 0 value. [this](https://github.com/Aaryan562/ivy/blob/min_stat_arg/ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py#L1703).

Reply:
As mentioned in the comment above, let's try generating the shapes for both `x` and `where` through the `mutually_broadcastable_shapes` helper. Thanks @Aaryan562 😄